scripts/package_target.sh

```diff
@@ -16,7 +16,7 @@ if [ $? != 0 ]; then
 fi
 
 if [ ${SKIP_CHROMIUM} = false ]; then
-    node scripts/download_chromium.js ${ARCH} ${OUT}
+   node scripts/download_chromium.js ${ARCH} ${OUT}
```

Comment:
```suggestion
   node scripts/download_chrome.js ${ARCH} ${OUT}
```
If you agree with my previous suggestion.